### ISSUE TITLE
issue with atlantis comment colors / spacing

### ISSUE THREAD

**Issue by mo-hit** [hootsuite/atlantis#137]
Seems to be related to a recent release. The PR comments should look something like
```diff
+ aws_eip.generic_eip.0
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"

+ aws_eip.generic_eip.1
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"

+ aws_eip.generic_eip.2
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"
```
and now my output looks like

```diff
+ aws_eip.generic_eip.0
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"

 + aws_eip.generic_eip.1
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"

 + aws_eip.generic_eip.2
    allocation_id:     "<computed>"
    association_id:    "<computed>"
    domain:            "<computed>"
    instance:          "<computed>"
    network_interface: "<computed>"
    private_ip:        "<computed>"
    public_ip:         "<computed>"
    vpc:               "true"
```
There seems to be a space thats been added somehow

**Comment by lkysow** [comment]
Terraform 0.10.* changed the output form of `plan`. See https://github.com/hashicorp/terraform/pull/15362/files

It's added 2 spaces before the `+`. We're just using the `diff` formatting of github as a hack to get coloured output so the extra 2 spaces messes it up.

I'll look through the other possible language highlighters to see if there's something that will still work. Could also change the output and delete the spaces but I'd prefer not to do that.

**Comment by mohit-gupta-hs1** [comment]
Fixed with terraform 0.10.5. They addressed it in https://github.com/hashicorp/terraform/pull/15311. Closing

**Comment by mo-hit** [comment]
Looks like this is fixed in terraform 0.10.5 by https://github.com/hashicorp/terraform/pull/15311
Sweet! 
Closing
